proto/buf/alpha/audit/v1alpha1/service.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.audit.v1alpha1;
+
+import "buf/alpha/audit/v1alpha1/event.proto";
+
+// AuditService is the Audit service.
+service AuditService {
```

Comment:
Not 100% certain now that this and the events definitions should live outside of `registry` pkg. Sounds like they make sense within it.

Reply:
I don't think we could make this `v1` while it depends on `registry/v1alpha1`. Unless we break that dependency, their versioning is tied and may as well live in the same package.

Reply:
https://linear.app/bufbuild/issue/BSR-666/reorganize-audit-code-and-pkgs